idaes/models_extra/gas_solid_contactors/properties/oxygen_iron_OC_oxidation/solid_phase_thermo.py

```diff
@@ -229,10 +302,10 @@ def define_metadata(cls, obj):
                     "method": "_dens_mass_particle",
                     "units": "kg/m3",
                 },
-                "cp_mol_comp": {"method": "_cp_mol_comp", "units": "kJ/mol.K"},
-                "cp_mass": {"method": "_cp_mass", "units": "kJ/kg.K"},
-                "enth_mass": {"method": "_enth_mass", "units": "kJ/kg"},
-                "enth_mol_comp": {"method": "_enth_mol_comp", "units": "kJ/mol"},
+                "cp_mol_comp": {"method": "_cp_mol_comp", "units": "J/mol.K"},
```

Comment:
Minor comment for reference - the "units" part is now redundant and not used, so you do not need to include it.

Reply:
Would you recommend removing "units" on all metadata lines, for the gas/solid/reaction modules?

Reply:
It doesn't hurt things if it is there, but it never actually gets used either.